llama_finetuning.py

```diff
@@ -94,6 +97,7 @@ def main(**kwargs):
             train_config.model_name,
             load_in_8bit=True if train_config.quantization else None,
             device_map="auto" if train_config.quantization else None,
+            use_cache=use_cache,
```

Comment:
So far, I have only tested this code path (not the `low_cpu_fsdp=True` path). If you guys have bandwidth, feel free to cherry pick this and test.